package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "poc-code-sydney",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1"
```

Comment:
Where are the packages that you've installed for axios, react-router-dom and react-icons?

Reply:
Reinstall:
![Screenshot 2020-04-14 at 20 06 39](https://user-images.githubusercontent.com/39728371/79213155-04d7e380-7e8c-11ea-9a47-c826034d6cd7.png)